torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -115,6 +119,45 @@ bool isSimpleMap(Node *node) {
   return true;
 }
 
+RegisterOperators reg_bn_unsqueeze({
+    Operator(
+        "aten::_ncf_expand(Tensor self, int ndim) -> Tensor",
```

Comment:
nit: _ncf_reshape might be a better name because we are reshaping instead of expanding

Reply:
It's as in NCHW, but it works with any dimension hence F for features instead of HW. I can rename it to reshape if you'd really want to, but I wouldn't like to block this PR on it if that's the only problem.